### PR TITLE
fix: remove volume icon masks

### PR DESCRIPTION
**Pull request by geurts**
Volume icons weren't rendering properly in XD due to spurious masks. I removed those and ran it through SVGOMG just in case. Seems to work now.

### CODE REVIEW

File: components/icons/images/tier1/volume-muted.svg [hunk 1]
```diff
@@ -1,8 +1,3 @@
...
-  </mask>
-  <g mask="url(#a)">
-    <path fill="#494c4e" d="M-1 0h18v18H-1z"/>
-  </g>
+<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
+  <path d="m1.585 1.19.097.079 4.905 4.577.41.382 3.775 3.523v-.005l1.628 1.517-.002.007 1.544 1.44.003-.008 1.373 1.28.097.079a1 1 0 0 0 .008.006l-.008.018 1.267 1.184a1 1 0 0 1-1.267 1.542l-.097-.08-.895-.835c-.278.434-.59.872-.936 1.312a.947.947 0 1 1-1.495-1.167c.393-.457.74-.938 1.044-1.44l-1.505-1.404a8.051 8.051 0 0 1-.954 1.249.953.953 0 0 1-1.338.019.947.947 0 0 1-.028-1.336c.347-.39.643-.818.883-1.273L7.002 8.97v5.286c0 .189-.111.36-.284.436a.424.424 0 0 1-.19.038.526.526 0 0 1-.321-.123l-2.915-2.719H.932a.93.93 0 0 1-.928-.928V7.132a.93.93 0 0 1 .928-.928h2.35l.378-.353-3.342-3.12A1 1 0 0 1 1.585 1.19zM13.44.827c.055.056 4.354 4.87 2.712 11.196l-1.662-1.55a10.72 10.72 0 0 0-2.452-8.369A.95.95 0 0 1 13.44.828zm-2.865 2.818a8.053 8.053 0 0 1 2.157 5.19L10.225 6.5a6.406 6.406 0 0 0-1.014-1.537.947.947 0 0 1 1.364-1.317z"/>
```
Comment (dlockhart) [inline components/icons/images/tier1/volume-muted.svg:2]: These look darker (black?) -- I think they need a ferrite fill:

```suggestion
  <path fill="#494c4e" d="m1.585 1.19.097.079 4.905 4.577.41.382 3.775 3.523v-.005l1.628 1.517-.002.007 1.544 1.44.003-.008 1.373 1.28.097.079a1 1 0 0 0 .008.006l-.008.018 1.267 1.184a1 1 0 0 1-1.267 1.542l-.097-.08-.895-.835c-.278.434-.59.872-.936 1.312a.947.947 0 1 1-1.495-1.167c.393-.457.74-.938 1.044-1.44l-1.505-1.404a8.051 8.051 0 0 1-.954 1.249.953.953 0 0 1-1.338.019.947.947 0 0 1-.028-1.336c.347-.39.643-.818.883-1.273L7.002 8.97v5.286c0 .189-.111.36-.284.436a.424.424 0 0 1-.19.038.526.526 0 0 1-.321-.123l-2.915-2.719H.932a.93.93 0 0 1-.928-.928V7.132a.93.93 0 0 1 .928-.928h2.35l.378-.353-3.342-3.12A1 1 0 0 1 1.585 1.19zM13.44.827c.055.056 4.354 4.87 2.712 11.196l-1.662-1.55a10.72 10.72 0 0 0-2.452-8.369A.95.95 0 0 1 13.44.828zm-2.865 2.818a8.053 8.053 0 0 1 2.157 5.19L10.225 6.5a6.406 6.406 0 0 0-1.014-1.537.947.947 0 0 1 1.364-1.317z"/>
```

File: components/icons/images/tier1/volume.svg [hunk 2]
```diff
@@ -1,8 +1,3 @@
...
-  </mask>
-  <g mask="url(#a)">
-    <path fill="#494c4e" d="M-1 0h18v18H-1z"/>
-  </g>
+<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
+  <path d="M7.003 3.835v10.421c0 .189-.112.36-.285.436a.424.424 0 0 1-.19.038.526.526 0 0 1-.321-.123l-2.915-2.719H.932a.93.93 0 0 1-.928-.928V7.132a.93.93 0 0 1 .928-.928h2.35l2.925-2.72a.483.483 0 0 1 .511-.084.478.478 0 0 1 .285.435zM13.44.828c.07.07 6.876 7.695.047 16.38a.947.947 0 1 1-1.495-1.167 10.73 10.73 0 0 0 .046-13.936A.95.95 0 0 1 13.44.828zm-2.865 2.818a8.053 8.053 0 0 1 2.162 5.4 8.053 8.053 0 0 1-2.16 5.4.953.953 0 0 1-1.338.019.947.947 0 0 1-.028-1.336 6.407 6.407 0 0 0 1.62-4.083 6.407 6.407 0 0 0-1.62-4.083.947.947 0 0 1 1.364-1.317z"/>
```
Comment (dlockhart) [inline components/icons/images/tier1/volume.svg:2]:
```suggestion
  <path fill="#494c4e" d="M7.003 3.835v10.421c0 .189-.112.36-.285.436a.424.424 0 0 1-.19.038.526.526 0 0 1-.321-.123l-2.915-2.719H.932a.93.93 0 0 1-.928-.928V7.132a.93.93 0 0 1 .928-.928h2.35l2.925-2.72a.483.483 0 0 1 .511-.084.478.478 0 0 1 .285.435zM13.44.828c.07.07 6.876 7.695.047 16.38a.947.947 0 1 1-1.495-1.167 10.73 10.73 0 0 0 .046-13.936A.95.95 0 0 1 13.44.828zm-2.865 2.818a8.053 8.053 0 0 1 2.162 5.4 8.053 8.053 0 0 1-2.16 5.4.953.953 0 0 1-1.338.019.947.947 0 0 1-.028-1.336 6.407 6.407 0 0 0 1.62-4.083 6.407 6.407 0 0 0-1.62-4.083.947.947 0 0 1 1.364-1.317z"/>
```

**Review by dlockhart — COMMENTED** [review]
I added some suggested changes to apply a ferrite fill, otherwise they end up looking black to me.

File: components/icons/images/tier1/volume.svg (hunk 2 above)
Comment (geurts) [reply components/icons/images/tier1/volume.svg:2]: Nice catch, thanks .. not sure how those got stripped

File: components/icons/images/tier1/volume-muted.svg (hunk 1 above)
Comment (geurts) [reply components/icons/images/tier1/volume-muted.svg:2]: thx!

**Review by dlockhart — APPROVED** [review]

**Pull request merged**